README.md

```diff
@@ -125,14 +125,14 @@ configured for the repository (see these [instructions](#prow-setup) for info on
   * The first step in the workflow should checkout the code using [checkout.sh](https://github.com/kubeflow/testing/tree/master/images/checkout.sh)
   * Code should be checked out to a shared NFS volume to make it accessible to subsequent steps
 1. Create a container to use with the Prow job
-  * For an example look at the [tensorflow/k8s](https://github.com/tensorflow/k8s/tree/master/test/test-infra) repository
+  * For an example look at the [kubeflow/testing](https://github.com/kubeflow/testing/blob/master/images/Dockerfile) repository
```

Comment:
For an example look at the [kubeflow/testing](https://github.com/kubeflow/testing/blob/master/images/Dockerfile) ~~repository~~ _Dockerfile_